01-factoring-overview.ipynb

```diff
@@ -390,10 +388,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from dwave.system.samplers import DWaveSampler\n",
+    "from dwave.system import DWaveSampler\n",
     "\n",
     "# Use a D-Wave system as the sampler\n",
-    "sampler = DWaveSampler(solver={'qpu': True})  "
+    "sampler = DWaveSampler() \n",
```

Comment:
This will work as expected only in Ocean 3.0.0+

Reply:
Yes, and we'd only want to drop pre-calculated embeddings for SDKs that have the improved chain-strength default behavior. This PR will come after https://github.com/dwave-examples/factoring-notebook/pull/10

Reply:
Merged now.